src/library/scala/collection/mutable/Iterable.scala

```diff
@@ -15,17 +15,7 @@ trait Iterable[A]
   * @define Coll `mutable.Iterable`
   */
 trait IterableOps[A, +CC[X], +C]
```

Comment:
Good point. I’m in favor of removing as many things as possible. The only drawback I can see is that it might confuse collection implementers to not have a `mutable.IterableOps` base trait to extend.

Reply:
Do we have other similar cases?

Reply:
We don’t have `mutable.LinearSeqOps`.

Reply:
Thanks! I'd remove it.

Reply:
Done!